db/views/analytics_cas_steps_v01.sql

```diff
@@ -0,0 +1,10 @@
+SELECT id,
+  referral_id,
+  name,
+  "order",
```

Comment:
Clean this to avoid sql keyword? Maybe
```suggestion
  "order" as sort_order,
```